### ISSUE TITLE
arcface loss 为都求 cos(target_logit + self.m2) 不是只有yi 求 cos(target_logit + self.m2) ， 而非本类求 cos(target_logit）？

### ISSUE THREAD

**Issue by cqray1990** [deepinsight/insightface#2513]
if self.m1 == 1.0 and self.m3 == 0.0:
            with torch.no_grad():
                target_logit.arccos_()
                logits.arccos_()
                final_target_logit = target_logit + self.m2
              #cos(target_logit + self.m2) 不是只有yi 求 cos(target_logit + self.m2) ， 而非本类求 cos(target_logit）？
                logits[index_positive, labels[index_positive].view(-1)] = final_target_logit
                logits.cos_()

**Comment by shaojiajun314** [comment]
他这个 是不是不太对啊啊，所有的 sample 同时加了 margin，我跑了下 自己的样本 完全不收敛啊

**Comment by shaojiajun314** [comment]
这个我跑了下，、不收敛，、

**Comment by shaojiajun314** [comment]
> 这个我跑了下，、不收敛，、

他这个 是不是对所有 sample 同时都加了 margin啊，怎么感觉 没看到 求和那一步
